core/src/model/ModelFactory.java

```diff
@@ -1,9 +1,16 @@
 package model;
 
+import model.enemies.Cyclops;
+import model.enemies.Mouse;
+
 public class ModelFactory {
     public static Model makeModel(String mapName) {
         Model model = new Model();
         model.initialize(new TiledMapLoader(mapName));
+        model.addEnemy(new Cyclops(100, 100, 1, 1, 1, model.getWorld()));
```

Comment:
Manually set into model i guess its only temporary, but if its temporary then why not do it in model? its gonna happen there the "correct way" anyway